src/BasisTheory.net/Reactors/Entities/ReactResponse.cs

```diff
@@ -0,0 +1,16 @@
+using System.Text.Json.Serialization;
+using Newtonsoft.Json;
+
+namespace BasisTheory.net.Reactors.Entities
+{
+    public class ReactResponse
+    {
+        [JsonProperty("tokens")]
+        [JsonPropertyName("tokens")]
+        public dynamic Tokens { get; set; }
+
+        [JsonProperty("raw")]
+        [JsonPropertyName("raw")]
+        public dynamic Raw { get; set; }
+    }
```

Comment:
Would it improve UX to make the response generic so that users can specify the types at time of request instead of having to deserialize to dynamic and then handle run-time conversion? E.g. 
```csharp
public class ReactResponse<T, R>
{
    [JsonProperty("tokens")]
    [JsonPropertyName("tokens")]
    public T Tokens { get; set; }

    [JsonProperty("raw")]
    [JsonPropertyName("raw")]
    public R Raw { get; set; }
}
```

Reply:
If we do this, it should be an overloaded option.